src/shared/SellerProfile/Body.js

```diff
@@ -279,11 +279,15 @@ const Body = (props) => {
           </ul>
         </Row>
         <Row title="Signed agreement" show={true}>
-          {!isEmpty(signed_agreements) && !isEmpty(head(signed_agreements)) && signed_agreements[0]['agreement'] &&
-          (<span><a href={signed_agreements[0]['agreement']['url']}>{signed_agreements[0]['agreement']['version']}
-          </a> signed on {signed_agreements[0]['signed_at']
-            && format(new Date(signed_agreements[0]['signed_at']), 'DD/MM/YYYY')}</span>)
-          }
+          {signed_agreements && signed_agreements.map((sa, i) => (
+            <React.Fragment key={i}>
+              {sa['agreement'] && <div>
+                <a href={sa['agreement']['url']}>{sa['agreement']['version']}</a>{' '}
+                signed on {sa['agreement']['signed_at'] && format(new Date(sa['agreement']['signed_at']), 'DD/MM/YYYY')}
+                {sa['signed_at'] && format(new Date(sa['signed_at']), 'DD/MM/YYYY')}
```

Comment:
Are there supposed to be two dates being output here? i.e. what happens if `sa['agreement']['signed_at']` and `sa['signed_at']` are both truthy?

Reply:
updated to only ever show one